dts/arc/yaml/intel,qmsi_ss-uart.yaml

```diff
@@ -0,0 +1,42 @@
+---
+title: Intel QMSI Uart
+id: intel,qmsi-uart
+version: 0.1
+
+description: >
+    This binding gives a base representation of the INTEL QMSI UART
+
+inherits:
+  - !include uart.yaml
+
+properties:
+  - compatible:
+      type: string
+      category: required
+      description: compatible strings
+      constraint: "intel,qmsi_ss-uart"
+
+  - reg:
+      type: array
+      description: mmio register space
+      generation: define
+      category: required
+
+  - interrupts:
+      type: array
+      category: required
+      description: required interrupts
+      generation: define
+
+  - pinctrl-\d+:
```

Comment:
remove clock & pin since you don't use any of that right now

Reply:
ok will remove it